src/executables/commdb.c

```diff
@@ -1369,6 +1389,14 @@ main (int argc, char **argv)
 	  commdb_Arg_ha_util_process_args = strdup (optarg);
 	  commdb_Arg_ha_start_util_process = true;
 	  break;
+	case COMMDB_SHUTDOWN_REVIVING_SERVER_S:
+	  commdb_Arg_shutdown_reviving_server = true;
+	  if (commdb_Arg_shutdown_reviving_server_name != NULL)
+	    {
+	      free_and_init (commdb_Arg_shutdown_reviving_server_name);
+	    }
+	  commdb_Arg_shutdown_reviving_server_name = strdup (optarg);
```

Comment:
가독성을 위해 바로 위의 case COMMDB_HA_START_UTIL_PROCESS_S:과 동일한 패턴으로 정리
```suggestion
	  if (commdb_Arg_shutdown_reviving_server_name != NULL)
	    {
	      free_and_init (commdb_Arg_shutdown_reviving_server_name);
	    }
	  commdb_Arg_shutdown_reviving_server_name = strdup (optarg);
	  commdb_Arg_shutdown_reviving_server = true;
```

Reply:
수정하였습니다.